newdle/api.py

```diff
@@ -7,7 +8,8 @@
 from marshmallow import fields
 from marshmallow.validate import OneOf
 from pytz import common_timezones_set
-from sqlalchemy.orm import selectinload
+from sqlalchemy import asc, desc, func
```

Comment:
some of the imports (e.g. `asc`) are unused (see the CI failures)

Reply:
Sorry, I accidentally missed that. I'm just about to force push.